README.md

```diff
@@ -277,34 +284,55 @@ const consumer = await memphisConnection.consumer({
       });
 ```
 
-Creating consumers with nestjs dependecy injection
+To set Up connection in nestjs
 
 ```js
-@Module({
-    imports: [MemphisModule.register()],
-})
-class ConsumerModule {
-    constructor(private memphis: MemphisService) {}
+async function bootstrap() {
+  const app = await NestFactory.createMicroservice<MicroserviceOptions>(
+    AppModule,
+    {
+      strategy: new MemphisServer({
+        host: 'localhost',
+        username: 'root',
+        connectionToken: 'memphis',
+      }),
+    },
+  );
+
+  await app.listen();
+}
+bootstrap();
+```
 
-    createConsumer() {
-        (async function () {
-                const consumer = await this.memphis.consumer({
-                    stationName: "<station-name>",
-                    consumerName: "<consumer-name>",
-                    consumerGroup: "",
-                });
-        })();
+To Consume messages in nestjs
+
+```js
+export class Controller {
+    import { consumeMessage } from 'memphis-dev/nest';
+
+    @consumeMessage({
+        stationName: '<station-name>',
+        consumerName: '<consumer-name>',
+        consumerGroup: ''
+    })
+    async signup(consumer) {
+            // On Message Success
+                console.log(message.getData().toString());
+                message.ack();
```

Comment:
where is the message comes from? it should be provided as a parameter of the signup function